### PR TITLE
Radio inputs, tabs, tags, wizard

### PR DESCRIPTION
**Pull request by brandonreid**
Updated styles for radio inputs, tabs, tags, and wizard.

### CODE REVIEW

File: src/components/Tag/PTag.vue [hunk 1]
```diff
@@ -37,29 +37,39 @@
...
-  text-sm
+  text-sm;
+  background-color: var(--p-color-tag-bg);
+  border-color: var(--p-color-tag-border);
+  color: var(--p-color-tag-text);
 }
 
-.p-tag__icon { @apply
+.p-tag__icon,
+.p-tag__icon svg { @apply
```
Comment (pleek91) [inline src/components/Tag/PTag.vue:63]: Seems like this `svg` selector shouldn't be needed if `p-icon` is working properly. Might be worth looking into.

**Review by pleek91 — APPROVED** [review]

File: src/components/Tag/PTag.vue (hunk 1 above)
Comment (brandonreid) [reply src/components/Tag/PTag.vue:63]: Yeah that's what I had figured too but didn't dig in, wasn't working properly in the tags demo. I'll go ahead and scope it out before merging this.

Comment (brandonreid) [reply src/components/Tag/PTag.vue]: So `p-icon` didn't have any sort of max width/height. I went ahead and added those in for that component. We'll just want to keep an eye out for any icons that get squeezed in the UI after the update.

Comment (brandonreid) [reply src/components/Tag/PTag.vue]: Ah you know what though.. Maybe `p-icon` shouldn't have a max width/height like that but enforce it's standard sizes.. `p-tag` is setting a width and height when it should just be setting the icon to `size="small"`. I'll update that real quick in the rebrand branch directly.

**Pull request merged**